typescript/sdk/src/middleware/liquidity-layer/LiquidityLayerApp.ts

```diff
@@ -208,12 +208,27 @@ export class LiquidityLayerApp extends HyperlaneApp<LiquidityLayerContracts> {
     console.debug(
       `Complete portal transfer for nonce ${message.nonce} on ${message.destination}`,
     );
-    await this.multiProvider.handleTx(
-      message.destination,
-      destinationPortalAdapter.completeTransfer(
-        utils.ensure0x(Buffer.from(vaa.vaaBytes, 'base64').toString('hex')),
-      ),
-    );
+
+    try {
+      await this.multiProvider.handleTx(
+        message.destination,
+        destinationPortalAdapter.completeTransfer(
+          utils.ensure0x(Buffer.from(vaa.vaaBytes, 'base64').toString('hex')),
+        ),
+      );
+    } catch (error: any) {
+      if (
+        error?.error?.reason &&
+        error.error.reason.includes('no wrapper for this token')
```

Comment:
```suggestion
        error?.error?.reason?.includes('no wrapper for this token')
```

Reply:
Does this actually work? I thought otherwise this ends up as `undefined.includes()` which throws an error?